app/views/account/comments/create.js.erb

```diff
@@ -0,0 +1,4 @@
+$('.comment').append($('<%= escape_javascript(render partial: 'account/comments/comment',
```

Comment:
hmm ... 
can you remove this empty div from html and just insert new comment HTML in .comments div? TRY